code/modules/admin/create_mob.dm

```diff
@@ -11,7 +11,7 @@
 	user << browse(create_panel_helper(create_mob_html), "window=create_mob;size=425x475")
 
 /proc/randomize_human(mob/living/carbon/human/H)
-	H.gender = pick(MALE, FEMALE)
+	H.gender = pick(MALE, FEMALE, PLURAL, NEUTER)
```

Comment:
This distrbution should also be weighted towards male and female since the character generator is setup to handle those better than the other 2 and can create better character models as a result.

Reply:
I second this. It will need pickweight list